std/fmt/index.zig

```diff
@@ -892,9 +897,150 @@ test "fmt.format" {
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value: u128 = 128;
+
+        const expected = try bufPrint(buf1[0..], "&value: u128@{x}", @ptrToInt(&value));
+        const actual = try bufPrint(buf2[0..], "&value: {*}", &value);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value: i2 = -1;
+
+        const expected = try bufPrint(buf1[0..], "&value: i2@{x}", @ptrToInt(&value));
+        const actual = try bufPrint(buf2[0..], "&value: {*}", &value);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value: i8 = -8;
+
+        const expected = try bufPrint(buf1[0..], "&value: i8@{x}", @ptrToInt(&value));
+        const actual = try bufPrint(buf2[0..], "&value: {*}", &value);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value: i127 = -127;
+
+        const expected = try bufPrint(buf1[0..], "&value: i127@{x}", @ptrToInt(&value));
+        const actual = try bufPrint(buf2[0..], "&value: {*}", &value);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value: i128 = -128;
+
+        const expected = try bufPrint(buf1[0..], "&value: i128@{x}", @ptrToInt(&value));
+        const actual = try bufPrint(buf2[0..], "&value: {*}", &value);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value = []u64 { 123 };
+
+        // Print Pointer Path1
+        const expected = try bufPrint(buf1[0..], "&value: [1]u64@{x}", @ptrToInt(&value));
+        const actual = try bufPrint(buf2[0..], "&value: {*}", &value);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        const value = ([][*]const u8){
+            c"one",
+            c"two",
+            c"three",
+        };
+
+        // Validate value[0] is a Pointer and Pointer.Size.Many so we use expected Path
+        const u32_ptr_info = @typeInfo(@typeOf(value[0]));
+        assert(TypeId(u32_ptr_info) == TypeId.Pointer);
+        assert(u32_ptr_info.Pointer.size == TypeInfo.Pointer.Size.Many);
+
+        // Print Pointer Path4
+        const expected = try bufPrint(buf1[0..], "value[0]: u8@{x}", @ptrToInt(value[0]));
+        const actual = try bufPrint(buf2[0..], "value[0]: {*}", value[0]);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value = []u16{1, 2};
+
+        // Print Pointer Path5
```

Comment:
what does this mean, "Path5"? there's no "path 5" for pointer printing

Reply:
Line 203.

Note: these "Print Pointer PathX" use to test PathX but don't do so now because of new Pointer state in format.

I've changed to comments to read:
```
// Before support for address printing using {*} this exercised Print Pointer PathX
```